src/test/java/com/wdtinc/mapbox_vector_tile/adapt/jts/MvtEncoderTest.java

```diff
@@ -38,9 +40,9 @@ public void singleLayer() throws IOException {
 
     @Test
     public void multipleLayers() throws IOException {
-        JtsLayer layer = new JtsLayer("Australia", PointGen.australia());
-        JtsLayer layer2 = new JtsLayer("United Kingdom", PointGen.uk());
-        JtsLayer layer3 = new JtsLayer("United States of America", PointGen.usa());
+        JtsLayer layer = new JtsLayer("Australia", PointGen.australia(), Extent.DEFAULT);
```

Comment:
Don't need to modify all the tests with a 3-arg if you remove the breaking API change to the constructors to the layer.